server/repositories/cmsQueries/__tests__/recentlyAddedContentQuery.spec.js

```diff
@@ -0,0 +1,94 @@
+const { getPagination } = require('../../../utils/jsonApi');
+const { RecentlyAddedContentQuery } = require('../recentlyAddedContentQuery');
+
+describe('Recently Added Content page query', () => {
+  let ESTABLISHMENTNAME;
+  let timeStamp;
```

Comment:
These are never reassigned and they are simply types, so should just be a constant:
```
const ESTABLISHMENTNAME = 'Wayland';
const TIMESTAMP = 1651737416;
```